components/content/Card.tsx

```diff
@@ -37,19 +37,24 @@ export const Card: React.FunctionComponent<PropsWithChildren<CardProps>> = ({hea
 
 export interface CollapsibleCardProps extends CardProps {
   startsCollapsed: boolean;
+  collapsedStateChanged?: (collapsed: boolean) => void;
```

Comment:
there wasn't a way to get notified when the collapsed state of a card changed - now there is